src/main/java/com/box/sdk/BoxSignRequestCreateParams.java

```diff
@@ -0,0 +1,454 @@
+package com.box.sdk;
+
+import java.util.List;
+
+import com.box.sdk.internal.utils.JsonUtils;
+import com.eclipsesource.json.JsonArray;
+import com.eclipsesource.json.JsonObject;
+
+/**
+ * Optional parameters for creating a Sign Request.
+ *
+ * @see BoxSignRequest
+ */
+public class BoxSignRequestCreateParams {
+
+    private Boolean isDocumentPreparationNeeded;
+    private String redirectUrl;
+    private String declinedRedirectUrl;
+    private List<BoxSignRequestRequiredAttachment> requiredAttachments;
+    private Boolean areAttachmentsEnabled;
+    private Boolean areTextSignaturesEnabled;
+    private Boolean isTextEnabled;
+    private Boolean areDatesEnabled;
+    private Boolean areEmailsDisabled;
+    private BoxSignRequestSignatureColor signatureColor;
+    private Boolean isPhoneVerificationRequiredToView;
+    private String emailSubject;
+    private String emailMessage;
+    private Boolean areRemindersEnabled;
+    private String name;
+    private List<BoxSignRequestPrefillTag> prefillTags;
+    private Integer daysValid;
+    private String externalId;
+
+    /**
+     * Gets the flag indicating if the sender should be taken into the builder flow to prepare the document.
```

Comment:
Can you please check the verbiage for these parameter comments against https://staging.developer.box.com/reference/resources/sign-requests/?